libsolidity/ast/Types.h

```diff
@@ -1154,14 +1154,13 @@ class FunctionType: public Type
 	/// of the parameters to false.
 	TypePointer copyAndSetGasOrValue(bool _setGas, bool _setValue) const;
 
-	/// @returns a copy of this function type where all return parameters of dynamic size are
-	/// removed and the location of reference types is changed from CallData to Memory.
-	/// This is needed if external functions are called on other contracts, as they cannot return
```

Comment:
Could we leave this in? I think that it supports the general understanding of what this function is doing.

Reply:
But this functionality is not part of the implementation anymore.